convert_zdf.py

```diff
@@ -0,0 +1,151 @@
+"""Convert ZDF point cloud to the available formats.
+
+Available formats:
+    .ply - Polygon File Format
+    .csv,.txt - [X, Y, Z, r, g, b, Contrast]
+    .png,.jpg,.bmp,.tiff - 2D RGB image
+"""
+
+import argparse
+from pathlib import Path
+import numpy as np
+import cv2
+import zivid
+
+
+def _options():
+    parser = argparse.ArgumentParser(
+        description="Convert from ZDF to your preferred format",
+        epilog="Example: $ python -u convert_zdf.py --csv --png Zivid3D.zdf Zivid3D_1.zdf",
```

Comment:
Is this the common way of writing usage?

Reply:
I don't know. I just thought it would be nice to add an example at the end. Would you rather have it before the arguments?